src/assets/widget.html

```diff
@@ -247,14 +247,7 @@ <h1 class="rs-user rs-small-headline">user@provider.com</h1>
   <div class="rs-box rs-box-error">
     <div class="rs-error-message"></div>
     <div class="rs-error-buttons">
-      <button class="rs-button rs-button-small rs-sync" title="Sync now">
-        <svg class="rs-icon rs-loop-icon" xml:space="preserve" version="1.1"
-             x="0px" y="0px" height="16" width="16" viewBox="0 0 512 512"
-             style="enable-background:new 0 0 512 512" xmlns="http://www.w3.org/2000/svg">
-          <path d="m273.4 300.5l-0.3 58c48.9-8.2 86.3-51 86.3-102.5 0-15.9-3.6-31-10-44.5-2.8-5.8-6-11.3-9.8-16.5l47.1-43.5c1.1 1.3 2.1 2.7 3.1 4 20.9 28 33.2 62.8 33.2 100.5v3.7c-1.5 71.5-47.6 132-111.4 154.6-12.3 4.3-25.2 7.3-38.5 8.7l-0.1 57-76.2-67-26.2-23 44.4-38.7 58.4-50.8z"/>
-          <path d="m89 252.3c1.6-72.1 48.3-133 112.9-155.2 11.7-4 24-6.8 36.8-8.1l0.1-57 76.1 66.9 26.2 23.1-44.3 38.6-58.4 50.9 0.2-57.9c-48.8 8.3-86 51.1-86 102.4 0 16 3.6 31.1 10.1 44.7 2.7 5.8 6 11.2 9.7 16.3l-47 43.6c-1.3-1.6-2.6-3.3-3.8-5-20.5-27.9-32.6-62.3-32.6-99.6v-3.7z"/>
-        </svg>
-      </button>
+      <a href="#" class="rs-reconnect hidden">Renew</a>
```

Comment:
The `hidden` here is unprefixed, but also, if that didn't cause any issues in testing, then it's likely not even needed, no?

Reply:
The commit that prefixed the remaining class names is newer than this PR. So this needs to be adapted after rebasing/merging.